INZFS.MVC/Controllers/FundApplicationController.cs

```diff
@@ -863,6 +863,21 @@ private ViewResult GetViewModel(Page currentPage, Field field)
                         multiSelect.UserInput = UserInputList;
                     }
                     return View("MultiSelectInput", PopulateModel(currentPage, model));
+                case FieldType.gdsAddressTextBox:
+                    model = PopulateModel(currentPage, new AddressInputModel(), field);
+                    var addressInputModel = (AddressInputModel)model;
+                    if (!string.IsNullOrEmpty(model.DataInput))
+                    {
+                        var userAddress = model.DataInput.Split(',').ToList();
+                        addressInputModel.addresslist = userAddress;
+                        addressInputModel.addresslist[0] = addressInputModel.AddressLine1;
+                        addressInputModel.addresslist[1] = addressInputModel.AddressLine2;
+                        addressInputModel.addresslist[2] = addressInputModel.City;
+                        addressInputModel.addresslist[3] = addressInputModel.County;
+                        addressInputModel.addresslist[4] = addressInputModel.City;
```

Comment:
City seems to be mentioned twice, is this expected

Reply:
var userAddress = model.DataInput.Split(',').ToList();
                        addressInputModel.AddressLine1 = userAddress[0];
                        addressInputModel.AddressLine2 = userAddress[1];
                        addressInputModel.City = userAddress[2];
                        addressInputModel.County = userAddress[3];
                        addressInputModel.PostCode = userAddress[4];

Reply:
This is an old commit